ADAL/src/ADRequestParameters.m

```diff
@@ -98,23 +98,44 @@ - (void)setRedirectUri:(NSString *)redirectUri
     _redirectUri = [redirectUri msidTrimmedString];
 }
 
+- (void)setScope:(NSString *)scope
+{
+    _scope = scope;
+}
+
+- (NSString *)openidScope
```

Comment:
Same as above. 
Perhaps we can return a set, openIDScopes?
We have a util to convert set to a string.